docs/release-notes-ps-8.4.4.md

```diff
@@ -0,0 +1,45 @@
+# Percona Distribution for MySQL 8.4.4 using Percona Server for MySQL (2025-02-)
+
+Percona Distribution for MySQL is the most stable, scalable, and secure open source MySQL distribution, with two download options: one based on Percona Server for MySQL and one based on Percona XtraDB Cluster. [Install Percona Distribution for MySQL](installing.md).
+
+This release is based on [Percona Server for MySQL 8.4.4-4](https://www.percona.com/doc/percona-server/8.4/release-notes/8.4.4-4.html) that includes all the features and bug fixes available in the [MySQL 8.4.4 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-4.html) and enterprise-grade features developed by Percona.
+
+## Release highlights
+
+### Percona Server for MySQL 8.4.4-4
+
+Improves the [Data masking](https://docs.percona.com/percona-server/8.4/data-masking-overview.md) performance by introducing an internal term cache. The new cache speeds up lookups for `gen_blocklist()` and `gen_dictionary()` functions by storing dictionary data in memory. However, if the dictionary table is modified directly (outside of the proper functions), the cache may become out of sync. To fix this, use the `new masking_dictionaries_flush()` function.
+
+Changes also affect row-based replication: dictionary changes on the source server are replicated, but the term cache on the replica doesn’t update immediately. To address this, a new system setting, `component_masking_functions.dictionaries_flush_interval_seconds()`, can be set to automatically refresh the cache at specified intervals, helping replicas stay in sync.
+
+Find more detailed information in the [Data masking overview](https://docs.percona.com/percona-server/8.4/data-masking-overview.md) and in the [Data masking component functions](https://docs.percona.com/percona-server/8.4/data-masking-function-list.md).
+
+### MySQL 8.4.4
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.4.4 and included in Percona Server for MySQL are the following:
+
+* Certain IO buffer serializations triggered an assertion in debug builds, causing the system to hang. (Bug #37139618)
+
+* Dropping the primary key and adding a new `AUTO_INCREMENT` column as the primary key in descending order using the `INPLACE` algorithm resulted in failure. (Bug #36658450)
+
+* Queries using a descending primary key with the `index_merge` optimization sometimes returned incorrect results, including missing rows. (Bug #106207, Bug #33767814)
+
+* When large transactions were being received and applied, and a request to stop the replication channel was made using STOP REPLICA, MySQL failed to stop the channel properly. As a result, it stopped processing channel commands. Additionally, the server shutdown process did not complete gracefully, requiring either the MySQL process to be killed or the host system to be restarted. (Bug #115966, Bug #37008345)
+
```

Comment:
* Fixed an assertion in debug builds where certain IO buffer serializations caused system hangs. (Bug #37139618)
* Resolved a failure when dropping the primary key and adding a new `AUTO_INCREMENT` column as the primary key in descending order using the `INPLACE` algorithm. (Bug #36658450)
* Corrected incorrect results, including missing rows, in queries that used a descending primary key with the `index_merge`* optimization. (Bug #106207, Bug #33767814)
* Addressed a replication channel issue where MySQL failed to stop the channel properly when large transactions were being processed, and STOP REPLICA was requested. This issue also prevented graceful server shutdown, requiring process termination or system restart. (Bug #115966, Bug #37008345)

Reply:
updated. thanks